interface/forms/patient_encounter/common.php

```diff
@@ -409,4 +415,18 @@ function duplicateVisit(enc, datestr) {
 ?>
 </script>
 
+<!-- Script for making fancybox compatible with jquery 3.1.1 as $.browser is removed in jquery 1.9 -->
+<script type="text/javascript">
+    jQuery.browser = {};
+    (function () {
+        jQuery.browser.msie = false;
+        jQuery.browser.version = 0;
+        if (navigator.userAgent.match(/MSIE ([0-9]+)\./)) {
+            jQuery.browser.msie = true;
+            jQuery.browser.version = RegExp.$1;
+        }
+    })();
+</script>
```

Comment:
It will not be needed everywhere but certainly at many places. Yeah, seems like a good idea. I would keep this in a different function in the headers file.